### PR TITLE
Add digitalio for steppers

### PR DESCRIPTION
**Pull request by caternuson**
For #43

Tested with RPI and a bipolar stepper motor. **NOT** tested with unipolar stepper (unavailable).

stepper_test

### CODE REVIEW

File: adafruit_motor/stepper.py [hunk 1]
```diff
@@ -57,9 +57,38 @@
 """Step a fraction of a step by partially activating two neighboring coils. Step size is determined
    by ``microsteps`` constructor argument."""
 
+_SINGLE_STEPS = (
```
Comment (ladyada) [inline adafruit_motor/stepper.py]: wanna make these bytearrays so they take less space? we have to cram this into crikit build for cpx

**Review by ladyada — CHANGES_REQUESTED** [review]

File: adafruit_motor/stepper.py (hunk 1 above)
Comment (caternuson) [reply adafruit_motor/stepper.py]: yah..good idea...

**Review by kattni — APPROVED** [review]
Looks good to me! Did not test. Thanks for the update!

**Comment by ladyada** [comment]
ok we'll see if this can fit! hope so :)

**Pull request merged**